src/test/java/core/basesyntax/service/FruitTransactionServiceImplTest.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.FruitTransactionDao;
+import core.basesyntax.dao.FruitTransactionDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.operation.BalanceOperationHandler;
+import core.basesyntax.service.operation.OperationHandler;
+import core.basesyntax.service.operation.PurchaseOperationHandler;
+import core.basesyntax.service.operation.ReturnOperationHandler;
+import core.basesyntax.service.operation.SupplyOperationHandler;
+import core.basesyntax.service.reader.ReaderServiceICsvImpl;
+import core.basesyntax.service.writer.WriterServiceCsvImpl;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FruitTransactionServiceImplTest {
+    private static final String NORMAL_FILE = "test2.csv";
+    private static final String BALANCE_RETURN_FILE = "test3.csv";
+    private String[] expectedResult;
+    private String[] resultExpected;
```

Comment:
Do we need both these parameters? They look like similar, and I think we could make them local